README.md

```diff
@@ -1,42 +1,124 @@
-# GenSX Monorepo
+# GenSX ⚡️
 
-This monorepo contains GenSX and its related packages - a framework for building LLM workflows and AI agents with JSX on the backend. Every GenSX component is a pure function, making it easily shareable by default.
+[![npm version](https://badge.fury.io/js/gensx.svg)](https://badge.fury.io/js/gensx)
+[![Website](https://img.shields.io/badge/Visit-gensx.dev-orange)](https://gensx.dev)
+[![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-## Repository Structure
+[GenSX](https://gensx.dev/overview/) is a simple typescript framework for building complex LLM applications. It’s built around functional, reusable components that are composed using JSX to create and orchestrate workflows.
 
-- `packages/` - Published packages
-- `examples/` - Example applications and use cases
-- `docs` - `https://gensx.dev` Documentation
-
-## Prerequisites
+## Why GenSX?
 
-- Node.js (LTS version recommended)
-- pnpm (v9.12.2 or later)
+- 🎯 **Pure Functions**: Components are pure TypeScript functions that are easily testable, reusable, and sharable
+- 🌴 **Natural Composition**: Chain LLM calls using JSX - a familiar, visual syntax that reads top-to-bottom like normal code
+- ⚡️ **Parallel by Default**: Components execute in parallel when possible while maintaining dependencies
+- 🔒 **Type-safe**: Full TypeScript support with no DSLs or special syntax - just standard language features
+- 🌊 **Streaming Built-in**: Stream responses with a single prop change, no refactoring needed
+- 🚀 **Built for Scale**: Start simple and evolve to complex patterns like agents and reflection without changing your programming model
 
-## Documentation
-
-For detailed documentation about GenSX, please refer check out the [docs](https://gensx.dev).
+Check out the [documentation](https://gensx.dev/basic-concepts) to learn more about building LLM applications with GenSX.
```

Comment:
Should this link to the overview page or the basic concepts to start?

Reply:
i'll update it to point to the overview; that is probably the best landing page for docs